lighthouse-core/gather/driver.js

```diff
@@ -28,12 +28,10 @@ const DEFAULT_PAUSE_AFTER_LOAD = 0;
 const DEFAULT_NETWORK_QUIET_THRESHOLD = 5000;
 // Controls how long to wait between longtasks before determining the CPU is idle, off by default
 const DEFAULT_CPU_QUIET_THRESHOLD = 0;
+// Controls maximum amount of time to wait before continuing
+const DEFAULT_MAX_WAIT_FOR_LOAD = require('../config/constants').MAX_WAIT_FOR_LOAD;
```

Comment:
we might not be able to do this now (lots of test modifications?), but it would be pretty cool to pull all the references to these constants out of driver altogether, and just have the various `pauseAfterLoadMs`, etc (basically all of `ConfigSettings`) required on the config by the time it gets into driver

Reply:
yeah sg for followup 👍